packages/redux/src/checkout/draftOrders/selectors.ts

```diff
@@ -0,0 +1,211 @@
+import { createSelector } from 'reselect';
+import { getEntities, getEntityById } from '../../entities/selectors/index.js';
+import { type StoreState } from '../../index.js';
+import buildQueryStringFromObject from '../../helpers/buildQueryStringFromObject.js';
+import type {
+  CheckoutOrder,
+  DraftOrder,
+  DraftOrdersQuery,
+} from '@farfetch/blackout-client';
+import type { DraftOrdersState } from './index.js';
+
+import {
+  getAllDraftOrders as getAllDraftOrdersFromReducer,
+  getDraftOrderCreations,
+  getDraftOrder as getDraftOrderFromReducer,
+  getRemoveDraftOrder as getRemoveDraftOrderFromReducer,
+  getUpdateDraftOrder as getUpdateDraftOrderFromReducer,
+} from './reducer.js';
+
+const getDraftOrdersEntities = (state: StoreState) =>
+  getEntities(state, 'draftOrders');
+
+/**
+ * Returns the loading status for the create draft order request for a specific draft order.
+ *
+ * @param state - Application state.
+ * @param orderId - Id of the checkout to get the loading status.
+ *
+ * @returns Boolean indicating if the request is loading or not.
+ */
+export const isDraftOrderCreationsLoading = (
+  state: StoreState,
+  orderId: CheckoutOrder['id'],
+) =>
+  !!getDraftOrderCreations(state.draftOrders as DraftOrdersState)[orderId]
+    ?.isLoading;
+
+/**
+ * Returns the error for the create draft order request for a specific draft order.
+ *
+ * @param state - Application state.
+ * @param orderId - Id of the checkout to get the create participation error.
```

Comment:
Please, check the descriptions to make sure they are correct. 

```suggestion
 * @param orderId - Id of the checkout order to get the create draft order request error.
```